src/js/modules/supervisors/HandleError.ts

```diff
@@ -48,30 +48,46 @@ export const validateDisableResponseCode = (
 };
 
 export const validateEnableResponseCode = (
-  responses: EnableCoprosReply
+  responses: EnableCoprosReply,
+  coprocessors: Coprocessor[]
 ): Error[][] => {
-  return responses.inputs.map((response) => {
+  return responses.inputs.map((response, index) => {
     const { id, response: codes } = response;
-    return codes.reduceRight<Error[]>((errors, code) => {
+    return codes.reduceRight<Error[]>((errors, code, topicIndex) => {
+      const coprocessor = coprocessors[index];
```

Comment:
What if `coprocessors.length` != `responses.length`?

Reply:
It's not possible, renpanda always response 1 topic 1 code response

Reply:
yeah but we should crash in situations like that, technically those are fatal errors and we develop quick. I could introduce a bug and an assertion here would make it obvious that an error was introduced in red panda. Especially if the assertions aren't on a hot path i think its a simple and good sanity check to add

Reply:
Got it, Done

Reply:
This remains to be addressed.

Reply:
My bad. It was fixed here: https://github.com/vectorizedio/redpanda/pull/182/files#diff-977ad8528d3f6c8aa4c61f49053098475a8f5987922654cbb047fa5ad10e150aR65